File: Entities/Intermediate/UserResponse.cs
using Core.Entities;

namespace Entities.Intermediate
{
    public class UserResponse : IEntity
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public int ResponseId { get; set; }
        public bool IsComplete { get; set; }
    }
}

File: WebUI/UI/src/app/datas/models/intermediate/user-response.ts
export class UserResponse {
  constructor(userId: number, responseId: number, isComplete: boolean) {
    this.userId = userId;
    this.responseId = responseId;
    this.isComplete = isComplete;
  }
  id: number;
  userId: number;
  responseId: number;
  isComplete: boolean;
}
